espanol/templo/templo.md

```diff
@@ -1,2 +1,6 @@
 Cuando estaban todos reunidos en las afueras del templo, el Consejero empezó a predicar como todas
 las tardes.
+
+[Alguien alzó la mano](manoArriba.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.